snapcraft/tests/test_meta.py

```diff
@@ -209,6 +209,34 @@ def test_wrap_exe_must_write_wrapper(self, mock_assemble_env):
 
         self.assertEqual(expected, wrapper_contents)
 
+    @patch('snapcraft.common.assemble_env')
+    def test_wrap_exe_writes_wrapper_with_basename(self, mock_assemble_env):
+        mock_assemble_env.return_value = """\
+PATH={0}/part1/install/usr/bin:{0}/part1/install/bin
+""".format(common.get_partsdir())
+
+        snapdir = common.get_snapdir()
+        os.mkdir(snapdir)
+
+        relative_exe_path = 'test_relexepath'
+        with open(os.path.join(snapdir, relative_exe_path), 'w'):
+            pass
```

Comment:
no it's not, I'll fix this in this test and the test I copied from, thanks for spotting this!